Error creating newspaper processes: No doctypes configured in opac.xml

The generation of newspaper processes fails with the error message "No doctypes configured in opac.xml". However, doctypes are no longer defined in the opac.xml, but in the ruleset.

Stack trace:
```
[ERROR] 2020-07-28 14:17:56.780 [http-nio-8080-exec-9] EmptyTask - No doctypes configured in opac.xml
org.kitodo.exceptions.DoctypeMissingException: No doctypes configured in opac.xml
    at org.kitodo.config.ConfigProject.getDocType(ConfigProject.java:98)
    at org.kitodo.config.ConfigProject.getAdditionalFields(ConfigProject.java:189)
    at org.kitodo.production.process.NewspaperProcessesGenerator.initializeTitleGenerator(NewspaperProcessesGenerator.java:395)
    at org.kitodo.production.process.NewspaperProcessesGenerator.initialize(NewspaperProcessesGenerator.java:311)
    at org.kitodo.production.process.NewspaperProcessesGenerator.nextStep(NewspaperProcessesGenerator.java:279)
    at org.kitodo.production.helper.tasks.GeneratesNewspaperProcessesThread.run(GeneratesNewspaperProcessesThread.java:80)
```

--- a/Kitodo/src/main/resources/kitodo_opac.xml
+++ b/Kitodo/src/main/resources/kitodo_opac.xml
@@ -12,6 +12,10 @@
  *
 -->
 <opacCatalogues>
+    <doctypes>
+        <type isPeriodical="false" isMultiVolume="false" rulesetType="" title="legacyDocType"/>
+    </doctypes>
+
     <catalogue title="GBV" description="Gemeinsamer Bibliotheksverbund">
         <interfaceType>sru</interfaceType>
         <returnFormat>xml</returnFormat>

--- a/Kitodo/src/main/resources/kitodo_projects.xml
+++ b/Kitodo/src/main/resources/kitodo_projects.xml
@@ -23,121 +23,51 @@
                     <select label="Fraktur">Fraktur</select>
                     <select label="anderes">anderes</select>
                 </item>
-                <!-- title for monograph and periodical -->
-                <item from="vorlage" isnotdoctype="multivolume" required="true" ughbinding="true" docstruct="topstruct"
+                <!-- title -->
+                <item from="vorlage" required="true" ughbinding="true" docstruct="topstruct"
                       metadata="TitleDocMain">Titel
                 </item>
-                <item from="vorlage" isnotdoctype="multivolume" required="true" ughbinding="true" docstruct="topstruct"
-                      metadata="TitleDocMainShort">Titel (Sortierung)
-                </item>
-                <!-- title  just for the multivolume -->
-                <item from="vorlage" isdoctype="multivolume" required="true" ughbinding="true" docstruct="topstruct"
-                      metadata="TitleDocMain">Titel
-                </item>
-                <item from="vorlage" isdoctype="multivolume" required="true" ughbinding="true" docstruct="topstruct"
+                <item from="vorlage" required="true" ughbinding="true" docstruct="topstruct"
                       metadata="TitleDocMainShort">Titel (Sortierung)
                 </item>
                 <!-- authors and creators-->
-                <item from="vorlage" isdoctype="monograph|multivolume|periodical" ughbinding="true"
+                <item from="vorlage" ughbinding="true"
                       docstruct="topstruct" metadata="ListOfCreators">Autoren
                 </item>
                 <!-- identifer -->
-                <item from="werk" isnotdoctype="periodical" ughbinding="true" metadata="TSL_ATS" docstruct="topstruct">
+                <item from="werk" ughbinding="true" metadata="TSL_ATS" docstruct="topstruct">
                     ATS
                 </item>
-                <item from="werk" isdoctype="periodical" ughbinding="true" metadata="TSL_ATS" docstruct="topstruct">
-                    TSL
-                </item>
-                <item from="vorlage" isdoctype="multivolume" required="true" ughbinding="true" docstruct="topstruct"
-                      metadata="CatalogIDSource">PPN analog c-Satz
-                </item>
-                <item from="werk" isdoctype="multivolume" required="true" ughbinding="true" docstruct="topstruct"
-                      metadata="CatalogIDDigital">PPN digital c-Satz
-                </item>
-                <item from="vorlage" isdoctype="monograph" required="true" ughbinding="true" docstruct="topstruct"
+                <item from="vorlage" required="true" ughbinding="true" docstruct="topstruct"
                       metadata="CatalogIDSource">PPN analog a-Satz
                 </item>
-                <item from="werk" isdoctype="monograph" required="true" ughbinding="true" docstruct="topstruct"
+                <item from="werk" required="true" ughbinding="true" docstruct="topstruct"
                       metadata="CatalogIDDigital">PPN digital a-Satz
                 </item>
-                <item from="vorlage" isdoctype="periodical" required="true" ughbinding="true" docstruct="topstruct"
-                      metadata="CatalogIDSource">PPN analog b-Satz
-                </item>
-                <item from="werk" isdoctype="periodical" required="true" ughbinding="true" docstruct="topstruct"
-                      metadata="CatalogIDDigital">PPN digital b-Satz
-                </item>
-                <item from="werk" isdoctype="periodical" required="true" ughbinding="true" docstruct="topstruct"
-                      metadata="ISSN">ISSN
-                </item>
-                <item from="vorlage" required="true" isdoctype="periodical" ughbinding="true" docstruct="firstchild"
-                      metadata="CatalogIDSource">PPN analog Band
-                </item>
-                <item from="werk" required="true" isdoctype="periodical" ughbinding="true" docstruct="firstchild"
-                      metadata="CatalogIDDigital">PPN digital Band
-                </item>
-                <!--title, number and authors for multivolumes and periodicals -->
-                <item from="vorlage" isdoctype="multivolume|periodical" required="true" ughbinding="true"
-                      docstruct="firstchild" metadata="TitleDocMain">Titel (Band)
-                </item>
-                <item from="vorlage" isdoctype="multivolume|periodical" required="true" ughbinding="true"
-                      docstruct="firstchild" metadata="TitleDocMainShort">Titel (Band) (Sortierung)
-                </item>
-                <item from="vorlage" isdoctype="multivolume" ughbinding="true" docstruct="firstchild"
-                      metadata="ListOfCreators">Autoren (Band)
-                </item>
-                <item from="vorlage" isnotdoctype="monograph" ughbinding="true" docstruct="firstchild"
-                      metadata="CurrentNo">Bandnummer
-                </item>
-                <item from="vorlage" isnotdoctype="monograph" ughbinding="true" docstruct="firstchild"
-                      metadata="CurrentNoSorting">Nummer (Sortierung)
-                </item>
-                <item from="vorlage" isdoctype="multivolume|periodical" required="true">Nummer (Benennung)</item>
-                <item from="vorlage" required="true" isdoctype="multivolume" ughbinding="true" docstruct="firstchild"
-                      metadata="CatalogIDSource">PPN analog f-Satz
-                </item>
-                <item from="werk" required="true" isdoctype="multivolume" ughbinding="true" docstruct="firstchild"
-                      metadata="CatalogIDDigital">PPN digital f-Satz
-                </item>
-                <!-- other metadata for all -->
-                <item from="vorlage" isdoctype="monograph|multivolume|periodical" ughbinding="true"
+                <!-- other metadata -->
+                <item from="vorlage" ughbinding="true"
                       docstruct="topstruct" metadata="PlaceOfPublication">Erscheinungsort
                 </item>
-                <item from="vorlage" isdoctype="monograph" ughbinding="true" docstruct="topstruct"
+                <item from="vorlage" ughbinding="true" docstruct="topstruct"
                       metadata="PublicationYear">Erscheinungsjahr
                 </item>
-                <item from="vorlage" isdoctype="periodical|multivolume" ughbinding="true" docstruct="firstchild"
-                      metadata="PublicationYear">Erscheinungsjahr
-                </item>
-                <item from="vorlage" isdoctype="multivolume|periodical" ughbinding="true" docstruct="firstchild"
-                      metadata="PublisherName">Verlag
-                </item>
-                <item from="vorlage" isdoctype="monograph" ughbinding="true" docstruct="topstruct"
+                <item from="vorlage" ughbinding="true" docstruct="topstruct"
                       metadata="PublisherName">Verlag
                 </item>
                 <item from="vorlage" ughbinding="true" docstruct="physSequence" metadata="shelfmarksource">Signatur</item>
-                <processtitle isdoctype="multivolume">TSL_ATS+'_'+CatalogIDDigital</processtitle>
-                <processtitle isdoctype="monograph">TSL_ATS+'_'+CatalogIDDigital</processtitle>
-                <processtitle isdoctype="periodical">TSL_ATS+'_'+CatalogIDDigital</processtitle>
-                <!-- <processtitle isnotdoctype="multivolume">TSL_ATS+'_'+CatalogIDDigital+'_'+CurrentNoSorting</processtitle> -->
                 <hide/>
             </itemlist>
             <opac use="true">
                 <catalogue>GBV</catalogue>
             </opac>
             <templates use="true"/>
-            <defaultdoctype>monograph</defaultdoctype>
+            <defaultdoctype>legacyDocType</defaultdoctype>
             <metadatageneration use="true"/>
         </createNewProcess>
         <tifheader>
-            <monograph>
+            <legacyDocType>
                 '|[[DOC_TYPE]]'+$Doctype+'|[[HAUPTTITEL]]'+TitleDocMain+'|[[AUTOREN/HERAUSGEBER]]'+Autoren+'|[[JAHR]]'+PublicationYear+'|[[ERSCHEINUNGSORT]]'+PlaceOfPublication+'|[[VERZ_STRCT]]'+TSL_ATS+'_'+CatalogIDDigital+'|'
-            </monograph>
-            <multivolume>
-                '|[[DOC_TYPE]]'+$Doctype+'|[[HAUPTTITEL]]'+TitleDocMain+'|[[AUTOREN/HERAUSGEBER]]'+Autoren+'|[[JAHR]]'+PublicationYear+'|[[ERSCHEINUNGSORT]]'+PlaceOfPublication+'|[[VERZ_STRCT]]'+TSL_ATS+'_'+CatalogIDDigital+'_'+Nummer (Benennung)+'|'
-            </multivolume>
-            <periodical>
-                '|[[DOC_TYPE]]'+$Doctype+'|[[HAUPTTITEL]]'+TitleDocMain+'|[[AUTOREN/HERAUSGEBER]]'+Autoren+'|[[JAHR]]'+PublicationYear+'|[[ERSCHEINUNGSORT]]'+PlaceOfPublication+'|[[VERZ_STRCT]]'+TSL_ATS+'_'+CatalogIDDigital+'_'+Nummer (Benennung)+'|'
-            </periodical>
+            </legacyDocType>
         </tifheader>
         <dmsImport/>
     </project>

